### ISSUE TITLE
AWS Opsworks Stack - Unable to set both default_os and configuration_manager_version

### ISSUE THREAD

**Issue by ldhertert** [hashicorp/terraform#5905]
My goal is to have a stack with the default OS being Ubuntu 14.04 LTS and using Chef 11.0.

Using the following code (partial)

```
resource "aws_opsworks_stack" "some_stack" {   
    default_os = "Ubuntu 14.04 LTS"        
    configuration_manager_name = "Chef"
    configuration_manager_version = "11.10"
    # ... remaining config
}
```

I receive the following error: `aws_opsworks_stack.some_stack: ValidationException: Default Os: Ubuntu 14.04 LTS is unsupported with Chef::11.4.`.  This leads me to believe that the configuration_manager_version setting is being ignored.

If I remove the default os property, the command runs fine and the end result is Ubuntu 12.04 LTS with Chef 11.10.  If I then add the default_os property back in, re-run terraform apply, then the stack is properly updated to the desired settings.

I am running terraform v0.6.14 on windows.

**Comment by smckim** [comment]
I believe that this was broken by #5724.  
After looking at the code, opsworks stacks look to be created with a lot of default values, and then updated with values specified in your configuration file after the initial creation (which is causing me issues trying to create Linux Chef 12 stacks). The default configuration_manager_version is 11.4, which is causing the failure with the inclusion of default_os in the initial stack creation (in which configuration_manager_version is not included). 
Downgrading to v0.6.13 should fix your issue.

**Comment by stack72** [comment]
Closed with #6244
